Backend/tests/test_class_ctrl.py

```diff
@@ -47,6 +47,19 @@ def test_getActiveClassList(self):
     self.assertNotEqual(response, None)
     print(response)
 
+  def test_getPublicClassList(self):
+    user =  "test@mentii.me"
+    response = class_ctrl.getPublicClassList(dynamodb, user)
+    self.assertNotEqual(response, None)
+    print(response)
```

Comment:
do you still want to keep these print statements? Also would write a more detailed test case that tests that the contents of the public class list is equal to something you expect